frontend/packages/ceph-storage-plugin/src/components/block-pool/states.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+  GrayUnknownIcon,
+} from '@console/shared/src/components/status/icons';
+
+enum ImageStates {
+  STARTING_REPLY = 'starting_replay',
+  STOP_REPLYING = 'stop_replaying',
+  REPLYING = 'replaying',
```

Comment:
Replay ??

Reply:
its REPLAYING i will change the key value is correct